### PR TITLE
[README] Add a leading dot to upToNextMinor

### PR DESCRIPTION
**Pull request by czechboy0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ To use swift-certificates, add the following dependency to your Package.swift:
 
 ```swift
 dependencies: [
-    .package(url: "https://github.com/apple/swift-certificates.git", upToNextMinor(from: "0.4.0"))
+    .package(url: "https://github.com/apple/swift-certificates.git", .upToNextMinor(from: "0.4.0"))
```
Comment (dnadoba) [inline README.md:36]:
```suggestion
    .package(url: "https://github.com/apple/swift-certificates.git", .upToNextMinor(from: "0.6.0"))
```

Comment (dnadoba) [inline README.md:36]:
```suggestion
    .package(url: "https://github.com/apple/swift-certificates.git", .upToNextMinor(from: "0.6.0"))
```

**Review by dnadoba — COMMENTED** [review]
Great catch! Lets increase to the latest released version as well and we can merge this.

**Review by Lukasa — APPROVED** [review]

**Comment by Lukasa** [comment]
@swift-server-bot add to allowlist

**Pull request merged**